index.html

```diff
@@ -8,38 +8,71 @@
 </head>
 
 <body>
-    <center>
-        <h1 style="margin-top: 20px; margin-bottom: 20px; ">
-           All awesome books
-        </h1>
-    <div class="displayWrap">
-        <div id="display_card" style="font-weight: bold;">
-
-        </div>
+    <header>
+        <nav class="nav">
+            <h1>All Awesome Books</h1>
+            <ul>
+                <li class="menuList"><a href="#">List</a></li>
+                <span style="display: inline-block; width:3px; background-color:rgb(0, 0, 0); color: black;"></span>
+                <li class="menuAdd"><a href="#">Add New</a></li>
+                <span style="display: inline-block; width:3px; background-color:rgb(0, 0, 0); color: black;"></span>
+                <li class="menuContact"><a href="#">Contact</a></li>
```

Comment:
- [OPTIONAL] I suggest using CSS classes instead of using inline styles to keep your code clean and follow the best practice 💯

> Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) 💡

![image](https://user-images.githubusercontent.com/45231276/205235420-b67d84a3-5429-4ff9-8ad0-9a5c6a76a05b.png)